indico/modules/events/registration/models/registrations.py

```diff
@@ -226,6 +249,12 @@ class Registration(db.Model):
             lazy=True
         )
     )
+    #: The registration tags assigned to this registration
+    registration_tags = db.relationship(
```

Comment:
It's useful to have backrefs on all relationships, and be explicit about the laziness. In this case, you can either make it `lazy=False` or alternative make it lazy but then make sure to specify the load option e.g. in the registration list to make sure we do not send separate queries for each registration to get its tags.

Also, just `tags` would be enough for the name - `registration.tags` is more concise than `registration.registration_tags`...